kolibri/core/analytics/api_urls.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework import routers
+
+from .api import PingbackNotificationDismissedViewSet
+from .api import PingbackNotificationViewSet
+
+router = routers.SimpleRouter()
+
+router.register(r'pingbacknotification', PingbackNotificationViewSet, base_name='pingbacknotification')
+router.register(r'pingbacknotificationdismissed', PingbackNotificationDismissedViewSet, base_name='pingbacknotificationdismissed')
```

Comment:
From a client API perspective, seems a little awkward to expose both DB tables. This seems like a server-side implementation detail.

Might be cleaner to have just a single endpoint rather than two:

* For a particular user, the notifications can be annotated with their dismissal status
* By default the notification collection can be filtered to only show non-dismissed items
* The user can dismiss a notification by setting the dismissal flag on a particular notification, which is mapped to the `PingbackNotificationDismissed` table in the serializer

Reply:
(cc @rtibbles this mirrors some comments you were making to me about how the logging API can be cleaned up by reducing the 1:1 model/API endpoint mapping we've traditionally used.